### PR TITLE
Detect missing dartino yaml

### PR DESCRIPTION
**Pull request by danrubel**
* add dartino.enable command used by dartino plugin to enable Dartino specific features
* prevent multiple notifications when creating dartino project
* add notification for non-project directories opened by the user
* warn user if they open subdirectory of Dart project
* move createDartinoYaml into dartino_util.dart
* notify users if Dartino project is missing dartino.yaml

@devoncarew

### CODE REVIEW

File: test/dartino_util_test.dart [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
```
Comment (devoncarew) [inline test/dartino_util_test.dart]: nit: date

**Comment by devoncarew** [comment]
lgtm

File: test/dartino_util_test.dart (hunk 1 above)
Comment (danrubel) [inline test/dartino_util_test.dart]: Done

**Comment by danrubel** [comment]
Merged https://github.com/dart-atom/dartlang/commit/aa5e92f736fd9eef2f719b45bdf3f129cc22859c

**Pull request closed without merging**